src/app/services/sms/__tests__/sms_count.server.model.spec.ts

```diff
@@ -230,9 +231,22 @@ describe('SmsCount', () => {
   })
 
   describe('VerificationCount Schema', () => {
+    const twilioMsgSrvcSid = smsConfig.twilioMsgSrvcSid
+
+    beforeAll(() => {
+      smsConfig.twilioMsgSrvcSid = 'all hail twilio, twilight of io'
+    })
+
+    afterAll(() => {
+      smsConfig.twilioMsgSrvcSid = twilioMsgSrvcSid
+    })
+
     it('should create and save successfully', async () => {
       // Arrange
       const smsCountParams = createVerificationSmsCountParams()
+      const expected = merge(smsCountParams, {
+        isOnboardedAccount: true,
```

Comment:
Why would this be true? Do you think this test should test for the default state, which is `isOnboardedAccount === false`

Reply:
agreed - our tests should have sensible defaults rather than having exceptional cases as being the default. changed accordingly!